src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/entities/Study.java

```diff
@@ -66,6 +66,10 @@
     @ManyToOne(optional = false)
     private Taxonomy taxonomy;
 
+    @JsonProperty(defaultValue = "false")
```

Comment:
If we are not going to display studies that have been deprecated, I found find more natural to annotate this with `@JsonIgnore`.

Reply:
I think if we put jsonignore we cant patch , if we wanna deprecate, and also we need to include a seperate endpoint to deprecate / undeprecate .

Reply:
OK! I have added in commit 8b0b4e7 a `access` property for `@JsonProperty` annotation. It is available from recent Jackson library. I have set it to `WRITE_ONLY` so that we can do `POST` and `PATCH` as @selvaebi mentioned, but also in the repsonse jsons of `GET` requests, the `deprecated` field will not appear. I think that is what @cyenyxe wanted, right?